pkg/agent/util/tc/tc_linux.go

```diff
@@ -0,0 +1,468 @@
+// Copyright 2021 Antrea Authors
```

Comment:
I don't know how others feel, but IMO it would have been great to have the TC stuff as a separate PR rather than submit one gigantic PR.

Reply:
The code of TC is just used by the implementation of NodePort/LoadBalancer, IMO, it seems a bit strange if the code in a separate PR.

Reply:
That's irrelevant IMO. As a rule, you should aim for small PRs. As long as the TC util package is self-contained, it can be (should be) in its own PR. Even if for now the AntreaProxyFull feature is the sole consumer of this package. However, for this PR, there is no need to split it at this stage since I can see you already had several rounds of reviews.

Reply:
@antoninbas, thanks for reminding.